addons/packages/pinniped/0.12.1/bundle/config/overlay/update_strategy_overaly.yaml

```diff
@@ -0,0 +1,41 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@overlay/match missing_ok=True,by=overlay.subset({"kind":"Deployment"})
```

Comment:
I do like the "why this file exists" we have in [overlay/update-crd-with-fallback-on-replace.yaml](https://github.com/vmware-tanzu/community-edition/tree/main/addons/packages/pinniped/0.12.1/bundle/config/overlay)

Reply:
Comments added